src/Components/OverlayTrigger/index.tsx

```diff
@@ -16,7 +17,7 @@ export class OverlayTrigger extends React.Component<OverlayTriggerProps> {
           {children}
         </Container>
 
-        <Overlay target={this.target} {...props}>
+        <Overlay {...props} target={() => findDOMNode(this.target)}>
           {cloneElement(overlay, {
```

Comment:
Is this related to a refs complaint?

Reply:
Yes, I believe that's where that error was coming from.